lib/virt_autotest/utils.pm

```diff
@@ -105,10 +105,23 @@ sub is_xen_host {
     return get_var("XEN") || check_var("SYSTEM_ROLE", "xen") || check_var("HOST_HYPERVISOR", "xen") || check_var("REGRESSION", "xen-hypervisor");
 }
 
+#check kernels by grep keywords from journals
+#welcome everybody to extend this function
+sub check_kernel {
+    my $machine //= 'localhost';
+
+    my @oops_keywords = ('Started Process Core Dump', 'Call Trace');
+    foreach my $keyword (@oops_keywords) {
+        my $cmd = "journalctl | grep '$keyword'";
+        $cmd = "ssh root\@$machine " . "\"$cmd\"" if $machine ne 'localhost';
+        record_soft_failure("$keyword in $machine fg. bsc#1204727") if script_run($cmd) == 0;
```

Comment:
Hi Julie, do I understand correctly that the current code logic is that once we find above oops_keywords in journal log, the test always record_soft_failure bsc#1204727? IMHO, the oops_keywords are general key words for coredump or call trace, but not necessarily this specific bug? And once such keywords found, the tester should check in details to see if NEW BUG needs to be opened?